monitors.yml

```diff
@@ -151,7 +151,28 @@ monitors:
       thresholds:
         critical: 2.0
         warning: 1.7
-    query: avg(last_5m):avg:system.load.1{*} by {host} > 2
+    query: avg(last_5m):avg:system.load.1{!host:bastion} by {host} > 2
+    type: metric alert
+
+  - message: |
+      {{#is_warning}}
+      System load over {{warn_threshold}}
+      {{/is_warning}}
+      {{#is_alert}}
+      System load over {{threshold}}!
+      {{/is_alert}}
+
+      @zxkuqyb@gmail.com
+      @slack-bonny-dd-alerts
+    multi: true
+    name: system load on bastion
+    options:
+      notify_no_data: false
+      require_full_window: true
+      thresholds:
+        critical: 3.0
+        warning: 2.7
+    query: avg(last_5m):avg:system.load.1{host:bastion} > 3
```

Comment:
This looks like we'd still be getting alerts for our current ansible usage that happens every 15/min, https://p.datadoghq.com/snapshot/view/dd-snapshots-prod/org_85238/2017-04-10/ae65ee2247372b3ff84fc5378c958d01466a1640.png
 
Maybe we can just use an average of more than 5m? The current spike  only lasts about 5/min so we can move to a 10 or 15min average?